### PR TITLE
Add windows move command

### PR DESCRIPTION
**Pull request by dgoemans**
#### Summary
Resolves this bug:
https://github.com/commercetools/merchant-center-application-kit/issues/2167

#### Description
Resolves and issue that prevents running `create-mc-app` on Windows

### CODE REVIEW

File: packages/create-mc-app/src/tasks/download-template.js [hunk 1]
```diff
@@ -65,13 +65,26 @@ module.exports = function downloadTemplate(options) {
             const sanitizedProjectDirectoryPath = options.projectDirectoryPath
               // Escape white spaces
               .replace(/ /g, '\\ ');
-            const result = await execa(
-              'mv',
```
Comment (emmenko) [inline packages/create-mc-app/src/tasks/download-template.js:69]: I suggest to simply add a ternary here:

```
process.platform == 'win32' || process.platform == 'cygwin' ? 'move' : 'mv'
```

**Review by emmenko — COMMENTED** [review]
Many thanks for reporting the issue and for providing a PR with the fix! 🙏

Can you check if this is the only place in the script that has incompatibility with windows?

File: packages/create-mc-app/src/tasks/download-template.js (hunk 1 above)
Comment (dgoemans) [reply packages/create-mc-app/src/tasks/download-template.js:69]: fair enough, initially i was using a totally different command with different params hence the if, but `move` is the same as `mv` so not needed anymore.

**Comment by dgoemans** [comment]
@emmenko This seems to be the only issue i've run into running this on Windows. If i bump into anything else, I'll create issues and send PRs :)

**Review by emmenko — APPROVED** [review]
Awesome, thanks again!

**Review by tdeekens — APPROVED** [review]
Thanks for your contribution!

**Comment by tdeekens** [comment]
@dgoemans you need to format your code using prettier.

**Comment by dgoemans** [comment]
Not just that, i've clearly been doing too much C# lately with a `==` instead of a `===`. It's a miracle i'm using single quotes for my strings 😆 
My bad... update incoming :)

**Review by emmenko — APPROVED** [review]

**Comment by dgoemans** [comment]
That's weird... the integration tests are failing very far from this. The only thing i noticed in the build log is 
```
[percy] Skipping visual tests - Missing Percy token
[percy] Running "yarn test:e2e:playground"
```
vs other previous PRs that do:
```
[percy] Percy has started!
[percy] Created build #3182: https://percy.io/commercetools-GmbH/app-kit-e2e-playground/builds/10122779
[percy] Running "yarn test:e2e:playground"
```
Any chance that's because it's building from my fork? I imagine it makes sense to protect project secrets from rando PRs 😄

**Comment by emmenko** [comment]
Yes that's the reason. I'll merge this PR into a separate branch and open a new PR to run CI again.

There is no easy way unfortunately to handle this kind of things in github actions at the moment.

**Comment by dgoemans** [comment]
👍 i'll keep an eye on the other PR and fix anything that comes out (altho i'm doubtful anything will pop up)
Cheers guys!

**Comment by emmenko** [comment]
@dgoemans the fix is released now: `@commercetools-frontend/create-mc-app@19.4.2`

Thanks again for the contribution 🙌

**Pull request merged**